mono/metadata/metadata.h

```diff
@@ -466,7 +466,7 @@ mono_method_header_get_num_clauses (MonoMethodHeader *header);
 MONO_API int
 mono_method_header_get_clauses (MonoMethodHeader *header, MonoMethod *method, void **iter, MonoExceptionClause *clause);
 
-MONO_API uint32_t 
+MONO_API MonoMarshalNative
```

Comment:
This does change the signedness. And C++ users using decltype() to duplicate the type and passing it around will get different name mangling. Both changes are probably ok, but I could err toward leaving the type as it was. 

There other ripple effects potentially like C++ users storing it in a uint32 previously. For absolute source and compat in the public interface, err toward leaving the type alone, and we insert casts elsewhere as needed. For likely easy to deal with breakage, and slightly increased type safety, change it.

Reply:
This change is optional.